### PR TITLE
Replace fs.exists with fs.stat

### PR DESCRIPTION
**Pull request by killtheliterate**

### CODE REVIEW

**Comment by killtheliterate** [comment]
Replaces fs.exists for new versions of node. Having a bit of trouble getting tests running yet.

File: src/index.sjs [hunk 1]
```diff
@@ -60,14 +60,20 @@ module.exports = function(fs) {
...
    * @method
-   * @summary Pathname → Future[Void, Boolean]
+   * @summary Pathname → Future[Error, Boolean]
    */
   exports.exists = exists;
   function exists(p) {
     return new Future(function(reject, resolve) {
-      fs.exists(p, function(a) {
-        resolve(a)
+
```
Comment (robotlolita) [inline src/index.sjs]: Could you remove this new line here?

File: src/index.sjs [hunk 2]
```diff
@@ -60,14 +60,20 @@ module.exports = function(fs) {
...
+      fs.stat(p, function(err) {
+        if (err) {
+          reject(err)
+        } else {
+          resolve(true)
+        }
       })
+
```
Comment (robotlolita) [inline src/index.sjs]: And this one here

**Comment by robotlolita** [comment]
Oh, sorry for taking so long to review things. There are no tests right now, so that's why it fails :)

Aside from the formatting issues pointed out, it looks good to me.

**Comment by killtheliterate** [comment]
Right on, I'll fix the newline stuff.

**Comment by robotlolita** [comment]
Thankies :)

**Pull request merged**